packages/epo-react-lib/src/atomic/Figure/Figure.tsx

```diff
@@ -11,12 +13,33 @@ interface FigureProps {
 const Figure: FunctionComponent<FigureProps> = ({
   children,
   caption,
+  layout = "vertical",
   withBackground,
   className,
 }) => (
-  <Styled.Figure $withBackground={withBackground} className={className}>
-    {children}
-    {caption && <Styled.FigCaption>{caption}</Styled.FigCaption>}
+  <Styled.Figure
```

Comment:
integrated the horizontal vs. vertical layouts into the Figure component + some refactor to remove passing props to styled-components